osu.Game/osu.Game.csproj

```diff
@@ -34,6 +34,11 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="ppy.ManagedBass" Version="2023.0.1" />
+    <PackageReference Include="ppy.ManagedBass.Enc" Version="3.0.0" />
+    <PackageReference Include="ppy.ManagedBass.Fx" Version="2023.0.0" />
+    <PackageReference Include="ppy.ManagedBass.Loud" Version="3.0.5" />
+    <PackageReference Include="ppy.osu.Framework.NativeLibs" Version="2024.0.1" />
```

Comment:
These cannot exist here, they are either provided by framework or need to be provided by framework (hence - move the code to o!f itself).

Reply:
These aren't really here for well, review. I committed these for the time being (as I'm still developing this) so that I can have the same references across my devices. I would've likely removed these changes prior to moving out of drafts.

Though, I do agree that they should mostly be provided by osu!f, especially considering my PR that provides those dll's. I'll move the references to o!f